op-node/rollup/derive/system_config.go

```diff
@@ -133,6 +136,33 @@ func ProcessSystemConfigUpdateLogEvent(destSysCfg *eth.SystemConfig, ev *types.L
 	case SystemConfigUpdateUnsafeBlockSigner:
 		// Ignored in derivation. This configurable applies to runtime configuration outside of the derivation.
 		return nil
+	case SystemConfigUpdateGasConfigEcotone:
+		if pointer, err := solabi.ReadUint64(reader); err != nil || pointer != 32 {
+			return NewCriticalError(errors.New("invalid pointer field"))
+		}
+		if length, err := solabi.ReadUint64(reader); err != nil || length != 8 {
+			return NewCriticalError(errors.New("invalid length field"))
+		}
+		packed := make([]byte, 8)
+		_, err := io.ReadFull(reader, packed)
+		if err != nil {
+			return NewCriticalError(errors.New("invalid packed scalars field"))
+		}
+		// confirm there is 32-8=24 bytes of 0-padding left
+		zeros := make([]byte, 24)
+		_, err = io.ReadFull(reader, zeros)
+		if err != nil {
+			return NewCriticalError(errors.New("didn't find expected padding"))
+		}
+		if !bytes.Equal(zeros, make([]byte, 24)) {
```

Comment:
Perhaps we define an internal variable that is all zeros to save an allocation here?

Reply:
done